### DISCUSSION TITLE
Realtime capture rate (G6)

### DISCUSSION THREAD

**Discussion by andyrozman** [NightscoutFoundation/xDrip#2660] [Q&A]
Is there a way to determine real time capture rate of G6 packets... I have switched phones and I am noticing that here and there I don't get the reading, but reading comes later with "Backsie". But I am still getting 100% capture rate, which doesn't make sense, because I know that reading is missed a lot of time (I am still trying to figure correct settings to work with my phone - I have OnePlus 8, and it seems to have very strict optimization rules, and disabling them doesn't really help)... 

So any Backsie, should NOT be counted into RT capture rate, and having option to see this would be helpful...  To determine how bad my new phone is behaving.

**Comment by Navid200** [comment]
I have wondered about this before.
@tzachi-dar @jamorham  Does "capture rate" include backfill?

**Reply by jonprogers** [reply]
In the 'Extra Status Line' settings there is a setting for 'Packet capture percentage' and 'Realtime packet capture percentage'. I think this second one gives you what you are looking for...

**Reply by andyrozman** [reply]
Thank you. That was excatly what I was looking for. My phone has about 61% RT capture, which is something I was expecting since I have seen a lot of times that entry was missed... 
Since its new phone, I have to run it through more of the differemt settings available to make it work little bit better.

**Reply by Navid200** [reply]
@jonprogers Thanks
